src/app/components/top-bar/top-bar.component.tsx

```diff
@@ -18,7 +18,7 @@ export class TopBar extends Component<IProps> {
     public getIcon(config) {
         const onClick = () => (config.action ? this.props.dispatch(config.action) : {});
         let icon = (
-            <TopAppBar.Icon navigation={!!config.action} onClick={onClick}>
+            <TopAppBar.Icon className="grey" navigation={!!config.action} onClick={onClick}>
```

Comment:
you should use `mdc-top-app-bar-icon-ink-color` mixin to change icon color.

Reply:
How can I do that? @include mdc-top-app-bar-icon-ink-color(grey); in .grey doesn't do the job it seems

Reply:
How is that done @include mdc-top-app-bar-icon-ink-color(grey) does not work in .grey

Reply:
it doesn't work because `mdc-top-app-bar-icon-ink-color` is a mixin for `Top-App-Bar` component, so you have to do the `@include...` in `.top-bar` class definition